packages/create/README.md

```diff
@@ -27,6 +27,13 @@ Calling `sap-ux add html` will add html files for local preview and testing to t
 sap-ux change add html ui5-test.yaml
 ```
 
+### new-model
```

Comment:
Could we just use 'add model`? I think it is clear that it is new.

Reply:
But what if you want to reserve this single word `model` for something else for the future? I am fine with both though but thinking about the future this could be a problem.

Reply:
`add model` is pretty clear to me, I wouldn't know what else we could do in the future that isn't adding a model :)
The only thing I could imagine is that we enhance the command to also work (but different) for Fiori apps.

Reply:
Renamed in: [9c969de](https://github.com/SAP/open-ux-tools/pull/2098/commits/9c969de2c305083b69db9312fff7bdcc4455a9e9)